dags/aps/parser.py

```diff
@@ -0,0 +1,136 @@
+                json_path="identifiers.arxiv",
+                extra_function=lambda x: {
+                    "value": re.sub("arxiv:", "", x, flags=re.IGNORECASE)
+                },
+            ),
+            NestedValueExtractor("abstract", json_path="abstract.value"),
+            NestedValueExtractor("title", json_path="title.value"),
+            CustomExtractor("authors", self._form_authors),
+            NestedValueExtractor("journal_title", json_path="journal.name"),
+            NestedValueExtractor("journal_issue", json_path="issue.number"),
+            NestedValueExtractor("journal_volume", json_path="volume.number"),
+            NestedValueExtractor(
+                "journal_year", json_path="date", extra_function=lambda x: int(x[:4])
+            ),
+            NestedValueExtractor("date_published", json_path="date"),
+            NestedValueExtractor(
+                "copyright_holder",
+                json_path="rights.copyrightHolders",
+                extra_function=lambda x: x[0]["name"],
+            ),
+            NestedValueExtractor("copyright_year", json_path="rights.copyrightYear"),
+            NestedValueExtractor(
+                "copyright_statement", json_path="rights.rightsStatement"
+            ),
+            NestedValueExtractor(
+                "licenses",
+                json_path="rights.licenses",
+                extra_function=lambda x: x[0]["url"],
+            ),
+            NestedValueExtractor("collections", json_path="HEP.Citeable.Published"),
+            CustomExtractor("field_categories", self._get_field_categories),
+            CustomExtractor("extra_data", self._build_files_data),
+        ]
+
+        super().__init__(extractors)
+
+    def _form_authors(self, article):
+        authors = article["authors"]
+        parsed_authors = []
+        for author in authors:
+            if author["type"] == "Person":
+                parsed_authors.append(
+                    {
+                        "full_name": author["name"],
+                        "given_names": author["firstname"],
+                        "surname": author["surname"],
+                        "affiliations": self._get_affiliations(
+                            article, author["affiliationIds"]
+                        ),
+                    }
+                )
+        return {"authors": parsed_authors}
+
+    def _get_affiliations(self, article, affiliationIds):
+        affiliations = article["affiliations"]
+        parsed_affiliations = []
+        for affiliation in affiliations:
+            if affiliation["id"] in affiliationIds:
+                value = affiliation["name"]
+                organization = (",").join(value.split(",")[:-1])
+                country = value.split(", ")[-1:]
+                parsed_affiliations.append(
+                    {"value": value, "organization": organization, "country": country}
+                )
+        return parsed_affiliations
```

Comment:
```suggestion
        affiliations = article["affiliations"]
        parsed_affiliations = [
            {"value": affiliation["name"], "organization": (",").join(value.split(",")[:-1]), "country": value.split(", ")[-1:]}
            for affiliation in article["affiliations"]
            if affiliation["id"] in affiliationIds
        ]
        return parsed_affiliations
```

Reply:
I couldn't commit this change, because it has some mistakes. However, I did a change regarding to this manually :) thank you